indico/modules/events/contributions/models/contributions.py

```diff
@@ -515,6 +515,9 @@ def enabled_editables(self):
         order = list(EditableType)
         return sorted(enabled_editables, key=lambda editable: order.index(editable.type))
 
+    def can_submit_editables(self, user):
+        return self.is_user_associated(user, check_abstract=True)
```

Comment:
Why `check_abstract=True`? Do we care about abstracts here?

Reply:
That way the user who submitted the abstract always has access, even if they are not listed as an author. We do the same e.g. for papers, and I think it makes sense: After all, if you submitted the abstract you may be the person who's "handling" that contribution, even if you are not an author/speaker yourself (but e.g. a secretary for the team who wrote it).